apps/frontend/src/components/DescriptionBox/DescriptionBox.tsx

```diff
@@ -113,7 +113,7 @@ export default function DescriptionBox({
           ) : null}
         </Col>
       </Row>
-      <Row className="mt-2 mt-md-4">
+      <Row className="mt-1 md-4">
```

Comment:
What did you intend to do here? `md-4` is not a valid bootstrap class.

Reply:
I have no idea what I was trying to do. Will remove it.